packages/shared/src/lib/auth.ts

```diff
@@ -295,6 +307,42 @@ export const logoutSession = (
     headers: { Accept: 'application/json', 'X-CSRF-Token': csrf_token },
   });
 
+export const initializeSettings = async (): Promise<InitializationData> => {
```

Comment:
Do we still need this?

Reply:
Should be not, I think. Let me remove this one quickly! Thanks for the fast review! 🚢